labelbox/data/serialization/ndjson/objects.py

```diff
@@ -21,6 +21,8 @@
 
 class NDBaseObject(NDAnnotation):
     classifications: List[NDSubclassificationType] = []
+    page: Optional[int] = None
```

Comment:
Wonder if these should be inside NDAnnotation. If we support PDFs for classifications in the future, we'd need to store the page/unit info inside the classification annotation